osm2pgsql.lua

```diff
@@ -474,6 +492,129 @@ function osm2pgsql.process_node(object)
     object.tags.wikidata == 'Q227569' or object.tags.wikidata == 'Q62172') then
         output_hstore['disputed_by'] = 'CN;RU;IN;GR'
     end
+-- Recast Northern Cyprus country label as region label for several POVs including China and Russia
+    if object.tags.place and object.tags.wikidata == 'Q23681' then
+        output_hstore['place'] = 'country'
```

Comment:
is this the right place tag? It's inconsistent with the comment

Reply:
Oh must have changed that at some point and not updated the comment. I'll fix it